core/src/main/kotlin/model/doc/TagWrapper.kt

```diff
@@ -19,7 +19,7 @@ sealed class NamedTagWrapper(root: DocTag, val name: String) : TagWrapper(root)
 }
 
 class Description(root: DocTag) : TagWrapper(root)
-class Author(root: DocTag) : TagWrapper(root)
+class Author(root: DocTag, name: String) : NamedTagWrapper(root, name)
```

Comment:
Do we really need this as a named Tag? In parser you just duplicate this property.

Reply:
There is possibility to process this tag later via `GroupedTags.withTypeUnnamed()`

Reply:
Not exactly, because `GroupedTags.withTypeUnnamed()`, as the name states, does grouping which in result gives you only one tag of specific type. This solution would fail if you had more than one author in KDoc (which is possible and justified). Named tag is grouped additionally by name.

Reply:
Of course @pikinier20, but this will lead to unwanted behaviour with expect/actuals, because name in expect will be treated as fallback. So in expect class you can have "Foo" and "Bar" authors, in actual have only "Bar", but both will be printed in generated documentation

Reply:
But how unnamed tag would avoid that while preserving possibility of adding multiple authors?